### ISSUE TITLE
Determine when the acquisition happened and therefore what FMV options are present

### ISSUE THREAD

**Issue by hickeng** [hickeng/financial#11]
The date quoted is 2023-11-22 but for the purposes of FMV assessment, was the acquisition processed between close on that date and trading on the 23rd? During trading on the 22nd? Between 21st and 22nd (f8937 uses close of the 21st for the FMV of fractional shares).

To determine FMV it's necessary to know the precise knowledge that was available to a hypothetical buyer/seller pair, which depends on precisely when the execution occurred.

Speculative options for FMV based on differing knowelege:
* During trading on 2023-11-22
  * Open: 983.69 
  * High: 987.99
  * Low: 971.00
  * Close: 972.00
  * Mean - $979.50

- [ ] can we use adjusted close (for splits and dividend and/or capital gain distributions) instead of quoted close?
- [ ] add the data needed to assess various options into the Reference sheet and add to the FMV dropdown

[Yahoo Finance market data for the period in question](https://ca.finance.yahoo.com/quote/AVGO/history?period1=1700438400&period2=1700870400&interval=1d&filter=history&frequency=1d&includeAdjustedClose=true)

**Comment by hickeng** [comment]
From costbasistools.com

> Comments on the merger of VMware Inc Class A common/Palo Alto CA-VMW-stock election at average price with Broadcom Inc common/San Jose CA-AVGO:

> Stock election by 96% of holders was oversubscribed and limited to 50 % of total merger consideration. Consideration prorated at 47.9% of $142.50 cash and 52.1% of 0.25200 AVGO common stock. AVGO market value of $979.50 based on average 11/22/2023 price per Form 8937 guidance. Use of 11/22/2023 closing price of $972.00 is also an acceptable IRS method.

**Comment by kevinbuhmann** [comment]
From the merger agreement:

>Unless the parties otherwise agree in writing, the closing of the transactions will occur on the third business day following the date on which all conditions to the transactions set forth in the merger agreement have been satisfied, or, to the extent permitted by applicable law, waived (other than those conditions that by their nature are to be satisfied at the closing of the transactions, but subject to the satisfaction or, to the extent permitted by applicable law, waiver of such conditions at the closing of the transactions). The first merger will be effective as of 11:59 p.m., New York City time, on the business day prior to the closing date. The second merger will be effective at 8:30 a.m., New York City time, on the closing date or at such other date and time as is agreed to by the parties and specified in the certificate of merger for the second merger.

https://www.sec.gov/Archives/edgar/data/1124610/000114036122035833/ny20005454x1_defm14a.htm

**Comment by hickeng** [comment]
@kevinbuhmann That's fantastic detail... and I've no idea what to do with it! When I wrote this issue I was expecting there to be a single point in time.

I suspect the only rational date to use is the 11:59pm one given after that there's no hypothetical possibility of shares changing hands.

I added the various different FMV options for the 22nd into the sheet a while back, with mean selected by default given that's the value quoted in Form8937.

I'm going to close this - more options than currently exist seems likely to just add confusion and the custom field should allow for it anyway.
